enterprise/web.jsf.navigation/src/org/netbeans/modules/web/jsf/navigation/graph/PageFlowSceneData.java

```diff
@@ -45,7 +45,7 @@ public class PageFlowSceneData {
      * Because the actual Page object is not necessarily the same, it uses the display name
      * as the key.
      * @param scene PageFlowScene
-     * @param utilities PageFlowUtilites
+     * @param utilities PageFlowUtilities
```

Comment:
That's not a proper description of that "utilities" variable, is it?

Reply:
yes: Utilit i es